coordinator/src/routes.rs

```diff
@@ -167,10 +166,6 @@ pub fn router(
         // pass registration
         .route("/api/register", post(post_register))
         .route("/api/users", post(post_register))
-        .route(
```

Comment:
I'd like to keep this endpoint as it's useful for debugging. I've moved it into `admin`

Reply:
A database query would also do, though.. :)